eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -9,7 +9,9 @@ function CreateReleases($pkgList, $releaseApiUrl, $releaseSha) {
 
     $releaseNotes = ""
     if ($pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent -ne $null) {
-      $releaseNotes = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent
+      [string]$releaseNotesTitle = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseTitle
+      [string]$releaseNotesContent = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent -Join [Environment]::NewLine
+      $releaseNotes = $releaseNotesTitle, $releaseNotesContent -Join [Environment]::NewLine
     }
```

Comment:
@scbedd Please have a look at this to make sure am not breaking anything. Basically am updating this to accommodate the new structure of the object returned from the Extract-ReleaseNotes script.